addons/web_editor/static/src/js/editor/odoo-editor/test/spec/editor.test.js

```diff
@@ -4034,6 +4034,35 @@ X[]
                     });
                 });
             });
+            describe("Links", () => {
+                it("should not split a link with role tab instead add the linebreak", async () => {
+                    await testEditor(BasicEditor, {
+                        contentBefore: '<p><a class="nav-link active o_default_snippet_text" href="#" role="tab">Ho[]me</a></p>',
```

Comment:
Do we need other classes like ```o_default_snippet_text```?

Reply:
No not as such, just kept it so that for an understanding that we tested a scenario for a link in a snippet, that has role tab/button etc. Should i remove `class` and simply keep `href` and `role` ?

Reply:
Yes, href and role is enough, maybe you need to keep class ```nav-link```